checkQC/app.py

```diff
@@ -1,27 +1,54 @@
 
 import sys
 
+import click
+
 from checkQC.qc_engine import QCEngine
-from checkQC.config import get_config
+from checkQC.config import get_config, get_handler_config
 from checkQC.run_type_recognizer import RunTypeRecognizer
 
 
-def start():
+@click.command("checkQC")
```

Comment:
A minor UX thing: Will you guys not get tired of having to switch case when writing this command?